src/vmecpp/cpp/vmecpp/vmec/output_quantities/output_quantities.cc

```diff
@@ -1258,7 +1258,8 @@ vmecpp::OutputQuantities vmecpp::ComputeOutputQuantities(
   output_quantities.vmec_internal_results.return_outputs_even_if_not_converged = indata.return_outputs_even_if_not_converged;
 
   if (vmec_status == VmecStatus::NORMAL_TERMINATION ||
-      vmec_status == VmecStatus::SUCCESSFUL_TERMINATION) {
+      vmec_status == VmecStatus::SUCCESSFUL_TERMINATION ||
+      output_quantities.vmec_internal_results.return_outputs_even_if_not_converged) {
```

Comment:
we can get this straight from the horse's mouth: 
```suggestion
      output_quantities.vmec_indata.return_outputs_even_if_not_converged) {
```

Reply:
Good point, fixed :)